packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -58,14 +72,43 @@ export class EditorSettingsSidebarComponent {
 	 * @param {EditorSidebarSection} sectionName Name of section.
 	 * @returns {Promise<void>} No return value.
 	 */
-	async expandSectionIfCollapsed( sectionName: EditorSidebarSection ): Promise< void > {
+	async expandSection( sectionName: EditorSidebarSection ): Promise< void > {
```

Comment:
Removes modality from function name following [this style guide rule](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/docs/style_guide.md#avoid-modal-verbs).